omod/pom.xml

```diff
@@ -57,6 +57,12 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
```

Comment:
It's not needed for compilation when building, contents in webModuleApplicationContext.xml are not compiled, it's only needed at runtime. There must be something you are missing somewhere